frontend/src/components/assessment-creation/CreateAssessmentHeader.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  Flex,
+  VStack,
+  Text,
+  Button,
+  Box,
+  Spacer,
+  HStack,
+  IconButton,
+} from "@chakra-ui/react";
+import {
+  TextOutlinedIcon,
+  SaveOutlinedIcon,
+  EyeOutlinedIcon,
+  MoreVerticalOutlineIcon,
+  ArrowBackOutlineIcon,
+} from "../../assets/icons";
+
+interface CreateAssessementHeaderProps {
+  assessmentName: string;
+  date: string;
+}
+
+const CreateAssessementHeader = ({
+  assessmentName,
+  date,
+}: CreateAssessementHeaderProps): React.ReactElement => {
+  return (
+    <Box
+      margin="-1.5em -2em 2em -2em"
+      padding="1.5em 2em 1.5em 2em"
+      borderBottom="1px"
+      borderBottomColor="grey.200"
+    >
+      <Flex minWidth="max-content">
+        <HStack spacing={6} alignItems="start">
+          <Button
+            size="sm"
+            color="blue.700"
+            leftIcon={<ArrowBackOutlineIcon />}
+          >
+            Back
+          </Button>
```

Comment:
Is there a reason why we shouldn't define an `onClick` function for the `Back` button? I think we should be able to redirect the user to the `Display Assessments` page!